src/components/CommentsCard.jsx

```diff
@@ -1,7 +1,42 @@
+import { fetchCommentsByArticle } from "../../utils/api";
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+
 export default function CommentsCard() {
+  const { article_id } = useParams();
+  const [comments, setComments] = useState();
+  const params = useParams();
+
+  useEffect(() => {
+    fetchCommentsByArticle(article_id).then((commentsData) => {
+      // console.log(typeof commentsData.comments);
+      setComments(commentsData.comments);
+    });
+  }, []);
+
+  console.log(params, "params");
+  console.log(comments, "comments");
+
+  if (!comments) {
+    return <div>Comment section loading, please wait.</div>;
+  }
+
   return (
     <>
       <h3>Comments Section</h3>
+
+      {comments.map((comment) => (
+        <div key={comment.comment_id}>
+          <h5>{comment.author}</h5>
```

Comment:
On this `comment.author` section, it might be worth making it slightly clearer to the user what it's mean to tell them, for example rather than just stating an author name, something like 'posted by <author>'